src/basics/BasicsValidator.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace basics;
+
+use InvalidArgumentException;
+
+class BasicsValidator implements BasicsValidatorInterface
+{
+
+    /**
+     * Implement the check functionality described in the method getMinuteQuarter (BasicsInterface Class) which throws Exception.
+     *
+     * Make sure the next PHPDoc instructions will be added:
+     * @param int $minute
+     * @throws InvalidArgumentException
+     */
+    public function isMinutesException(int $minute): void
+    {
+        if($minute<0 || $minute>60){
```

Comment:
Please refactor it according to the [PSR-12](https://www.php-fig.org/psr/psr-12/)